testdata/grpc_testing_not_regenerate/simple.proto

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package grpc.testdata.grpc_testing_not_regenerate;
+
+option go_package = "google.golang.org/grpc/testdata/grpc_testing_not_regenerate";
+
+// SimpleMessage is used to hold string data.
+message SimpleMessage {
```

Comment:
I feel it is better to name the test message as `MessageV1Only`, and give a short comments describing why it must not be regenerated.

Reply:
Sorry I missed that.